argo-cd-apps/base/member/infra-deployments/kubearchive/kubearchive.yaml

```diff
@@ -13,9 +13,27 @@ spec:
               values:
                 sourceRoot: components/kubearchive
                 environment: staging
-                clusterDir: ""
+                clusterDir: base
           - list:
-              elements: []
+              elements:
+                - nameNormalized: stone-stage-p01
+                  values.clusterDir: stone-stage-p01
+                # Private
+                - nameNormalized: kflux-ocp-p01
+                  values.clusterDir: kflux-ocp-p01
+                - nameNormalized: stone-prod-p01
+                  values.clusterDir: stone-prod-p01
+                - nameNormalized: stone-prod-p02
+                  values.clusterDir: stone-prod-p02
+                # Public
+                - nameNormalized: stone-prd-m01
+                  values.clusterDir: stone-prd-m01
```

Comment:
```suggestion
```

m01 is deprecated and it will be removed soon. @hugares should skip it in this rollout already?

Reply:
I removed it, I can add it back if needed.